PR review for Ecommerce App

saurav49 opened the pull request:
Name: Saurav Biswas
Link: https://barakmart.netlify.app
You can use the following credentials, username: admin, password: password123
OR you can also SignUp and create your own account.

saurav49 committed:
chore

added all files

saurav49 committed:
refractor

Xdev200 commented on src/components/cart/Cart.js:
```diff
@@ -0,0 +1,54 @@
+
+  return (
+    <div
+      style={{ display: "flex", flexDirection: "column", alignItems: "center" }}
```
Avoid inline styles

Xdev200 commented on src/components/navbar/LogOutButton.js:
```diff
@@ -0,0 +1,33 @@
+import React from "react";
```
remove import react

Xdev200 commented on src/components/productList/AddToCartButton.js:
```diff
@@ -0,0 +1,76 @@
+      : checkIsInCartList(state.cartList, productId)
+      ? updateCartQuantity({
+          url: `/api/cartLists`,
+          listType: "cartList",
```
move logic to custom hook

saurav49 committed:
refractor
few inline style removed

saurav49 committed:
Update README.md

saurav49 committed:
style
login page style changes

saurav49 committed:
refractor

refractor all auth code

saurav49 committed:
refractor

refractor all auth code

saurav49 committed:
refractor

refractor auth code

saurav49 committed:
chore

added productDetail page

saurav49 committed:
chore

added landing page(Home), category page, footer

saurav49 committed:
Merge branch 'development' of https://github.com/saurav49/TheBarakMart-An-Ecommerce-website into development

saurav49 committed:
refractor

saurav49 committed:
refractor

saurav49 committed:
remove favicon

saurav49 committed:
refractor AuthContext

saurav49 committed:
refractor

new methods added in datacontext, refractor of code in reducer function, props added in addtocartbtn, wishlist btn

saurav49 committed:
refractor

saurav49 committed:
update readme and refractor code

saurav49 committed:
refractor

saurav49 committed:
refractor-detailProdductPage

saurav49 committed:
gitignore added

saurav49 committed:
update readme.md

saurav49 committed:
chore

saurav49 committed:
chore- final checkout created

saurav49 committed:
chore

saurav49 committed:
refractor

saurav49 committed:
refractor - multiple address user call fixed

saurav49 committed:
refractor - console removed

saurav49 committed:
issue fix

saurav49 committed:
fix-netlify 404 on page reload

saurav49 committed:
create favicon.ico

saurav49 committed:
fixes and enhancement

- new card component created
- unnecessary console removed
- token added to all api calls

saurav49 committed:
fix-category page

- update wishlist status and cart status

saurav49 committed:
features added

- user profile section added
- orders section added
- edit address

saurav49 committed:
fix

saurav49 committed:
style-order page

saurav49 committed:
chore

- order successful toast created
